sopel/modules/reddit.py

```diff
@@ -431,7 +431,7 @@ def get_channel_spoiler_free(bot, trigger):
         bot.say('%s is flagged as spoilers-allowed' % channel)
 
 
-@rule(r'.*(?<!\S)/?(?P<prefix>r|u)/(?P<id>[a-zA-Z0-9-_]+)\b.*')
+@find(r'.*(?<!\S)/?(?P<prefix>r|u)/(?P<id>[a-zA-Z0-9-_]+)\b.*')
```

Comment:
Do you still need the `.*` to prefix and suffix the pattern? Can't this be simplified to something like `(?P<prefix>r|u)/(?P<id>[a-zA-Z0-9-_]+)`? I didn't dive into the issue so maybe there is a smarter regex to find here.

Reply:
Good call, that was one big reason for wanting `find` rules in the first place. 🤦‍♂️